_posts/2021-02-23-OpenSSL.markdown

```diff
@@ -0,0 +1,58 @@
+---
+title: Switching back to OpenSSL
+layout: post
+---
+
+The Void Linux team is planning on [performing the switch to
```

Comment:
This needs a date.  Also we're not planning on doing it, we will be performing the switch.

Reply:
The Void Linux team is [switching back to OpenSSL](void-linux/void-packages/pull/21056)